pkg/scaffold/scaffold.go

```diff
@@ -31,7 +31,7 @@ import (
 
 	"sigs.k8s.io/kubebuilder/pkg/model"
 	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
-	"sigs.k8s.io/kubebuilder/pkg/scaffold/project"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/util"
```

Comment:
IHMO: te should be utils in the plural.

Reply:
Lets keep the discussion in https://github.com/kubernetes-sigs/kubebuilder/pull/1313#discussion_r367866962